src/ontology/reports/README.md

```diff
@@ -0,0 +1,54 @@
+## Codebook: `src/ontology/reports/`
```

Comment:
Added `reports/README.md`. I also have this in another PR. I'll handle the merge conflict though when I rebase after whichever one is merged first.